### PR TITLE
Improves Table-ing Logging

### PR DESCRIPTION
**Pull request by bobbah**
:cl: bobbahbrown
tweak: Getting tabled is now far easier to prove due to advancements in logging technology!
/:cl:

Old table logs suck!

`[2018-10-21 07:09:38.492] ATTACK: FunnyMan2003/(Yanik Pulkkinen) has pushed 2Haku/(Len Riker) (NEWHP: 98.4)  (Central Primary Hallway (132, 109, 2))`

New table logs rock!

`[2018-10-27 12:42:51.038] ATTACK: Bobbahbrown/(Joel Judge) has tabled *no key*/(monkey (542)) onto the table (NEWHP: 100)  (Xenobiology Lab (169, 103, 2))
`

Don't delay, merge today!

### CODE REVIEW

**Comment by Jordie0608** [comment]
You're passing a string to an argument that expects an atom reference. This is bad.

**Comment by bobbahbrown** [comment]
> You're passing a string to an argument that expects an atom reference. This is bad.

checked, wrekt, and no longer suspect.

`log_combat(user, pushed_mob, "pushed", null, "onto [src]")`

**Comment by optimumtact** [comment]
did the log still come out fine?

File: code/game/objects/structures/tables_racks.dm [hunk 1]
```diff
@@ -126,7 +126,7 @@
 	pushed_mob.Paralyze(40)
 	pushed_mob.visible_message("<span class='danger'>[user] pushes [pushed_mob] onto [src].</span>", \
 								"<span class='userdanger'>[user] pushes [pushed_mob] onto [src].</span>")
-	log_combat(user, pushed_mob, "pushed", "their hands onto [src]")
+	log_combat(user, pushed_mob, "pushed", null, "onto [src]")
```
[the remark this answers is not in the document]
Comment (bobbahbrown) [reply code/game/objects/structures/tables_racks.dm]: resolved in c2a9bde

**Comment by bobbahbrown** [comment]
> did the log still come out fine?

Yeah! I've been updating the OP with the current log message produced.

`[2018-10-27 12:42:51.038] ATTACK: Bobbahbrown/(Joel Judge) has tabled *no key*/(monkey (542)) onto the table (NEWHP: 100)  (Xenobiology Lab (169, 103, 2))`

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**